src/hotspot/share/jeandle/jeandleAbstractInterpreter.cpp

```diff
@@ -1580,14 +1600,22 @@ void JeandleAbstractInterpreter::arith_op(BasicType type, Bytecodes::Code code)
   }
 }
 
+// Call a Java operation, without exception handling.
 llvm::CallInst* JeandleAbstractInterpreter::call_java_op(llvm::StringRef java_op, llvm::ArrayRef<llvm::Value*> args) {
   llvm::Function* java_op_func = _module.getFunction(java_op);
   assert(java_op_func != nullptr, "invalid JavaOp");
-  llvm::CallInst* call_inst = _ir_builder.CreateCall(java_op_func, args);
-  call_inst->setCallingConv(llvm::CallingConv::Hotspot_JIT);
+  llvm::CallInst* call_inst = call_jeandle_routine(java_op_func, args, llvm::CallingConv::Hotspot_JIT);
```

Comment:
According to the comment, `call_jeandle_routine` is used to call a JeandleRuntimeRoutine, but we use it to call a JavaOp here. This may lead to some misunderstandings.
The same as `call_java_op_ex`.